client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss

```diff
@@ -1,25 +1,25 @@
 @import 'assets/stylesheets/gutenberg-base-styles';
 @import '../../mixins';
+@import '../../variables.scss';
 
 .acquire-intent {
 	@include onboarding-font-recoleta;
-	align-items: center;
 	background-color: var( --contrastColor );
 	color: var( --mainColor );
-	display: flex;
 	font-size: 64px;
-	height: 100%;
-	justify-content: center;
-	left: 0%;
-	position: fixed;
 	tab-size: 4;
-	top: 0;
 	transition: color 0.5s ease-in-out 1s, background-color 0.5s ease-in-out 1s;
-	width: 100%;
-}
 
-.acquire-intent__questions {
-	max-width: 1200px;
+	@media ( min-width: 600px ) {
```

Comment:
Maybe this also needs to move to `variables` as the `mobile-breaking-point`.

Reply:
Good catch! Actually we already have [breakpoints](https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#media-queries) in Calypso styles so I'm switching to that and we're going to use 660px instead 600px.

Reply:
we can use breakpoints mixins from [`@wordpres/base-styles`](https://github.com/WordPress/gutenberg/tree/e16260ff98ee4c3163332d7659b50c4fc7a78320/packages/base-styles) package, which we import to Gutenboarding:

https://github.com/Automattic/wp-calypso/blob/2161f3032d5ec6f7d63ef64384958a410774d5d7/client/assets/stylesheets/gutenberg-base-styles.scss#L4-L5

https://github.com/Automattic/wp-calypso/blob/2161f3032d5ec6f7d63ef64384958a410774d5d7/client/landing/gutenboarding/style.scss#L4

Reply:
Great! 600px it is then 🎯

Reply:
Right but maybe with `break-small` mixin? :-) 

https://github.com/WordPress/gutenberg/blob/e16260ff98ee4c3163332d7659b50c4fc7a78320/packages/base-styles/_mixins.scss#L35

Not a biggie tho and can be merged without.